code/modules/client/verbs/ooc.dm

```diff
@@ -100,9 +97,12 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 	set name = "Set Your OOC Color"
 	set category = "Preferences"
 
+	/*
+	Not sure how analy retarded you made this exactly TG.
```

Comment:
w/e the point is don't give it to everyone, just lock it behind admin *for now* we can do fancy patreon nonsense later.